plugin/telescope.lua

```diff
@@ -0,0 +1,144 @@
+
+  -- Used for highlighting the matched line inside Previewer. Works only for (vim_buffer_ previewer)
+  TelescopePreviewLine = { default = true, link = "Visual" },
+  TelescopePreviewMatch = { default = true, link = "Search" },
+
+  TelescopePreviewPipe = { default = true, link = "Constant" },
+  TelescopePreviewCharDev = { default = true, link = "Constant" },
+  TelescopePreviewDirectory = { default = true, link = "Directory" },
+  TelescopePreviewBlock = { default = true, link = "Constant" },
+  TelescopePreviewLink = { default = true, link = "Special" },
+  TelescopePreviewSocket = { default = true, link = "Statement" },
+  TelescopePreviewRead = { default = true, link = "Constant" },
+  TelescopePreviewWrite = { default = true, link = "Statement" },
+  TelescopePreviewExecute = { default = true, link = "String" },
+  TelescopePreviewHyphen = { default = true, link = "NonText" },
+  TelescopePreviewSticky = { default = true, link = "Keyword" },
+  TelescopePreviewSize = { default = true, link = "String" },
+  TelescopePreviewUser = { default = true, link = "Constant" },
+  TelescopePreviewGroup = { default = true, link = "Constant" },
+  TelescopePreviewDate = { default = true, link = "Directory" },
+  TelescopePreviewMessage = { default = true, link = "TelescopePreviewNormal" },
+  TelescopePreviewMessageFillchar = { default = true, link = "TelescopePreviewMessage" },
+
+  -- Used for Picker specific Results highlighting
+  TelescopeResultsClass = { default = true, link = "Function" },
+  TelescopeResultsConstant = { default = true, link = "Constant" },
+  TelescopeResultsField = { default = true, link = "Function" },
+  TelescopeResultsFunction = { default = true, link = "Function" },
+  TelescopeResultsMethod = { default = true, link = "Method" },
+  TelescopeResultsOperator = { default = true, link = "Operator" },
+  TelescopeResultsStruct = { default = true, link = "Struct" },
+  TelescopeResultsVariable = { default = true, link = "SpecialChar" },
+
+  TelescopeResultsLineNr = { default = true, link = "LineNr" },
+  TelescopeResultsIdentifier = { default = true, link = "Identifier" },
+  TelescopeResultsNumber = { default = true, link = "Number" },
+  TelescopeResultsComment = { default = true, link = "Comment" },
+  TelescopeResultsSpecialComment = { default = true, link = "SpecialComment" },
+  TelescopeResultsFileIcon = { default = true, link = "Normal" },
+
+  -- Used for git status Results highlighting
+  TelescopeResultsDiffChange = { default = true, link = "DiffChange" },
+  TelescopeResultsDiffAdd = { default = true, link = "DiffAdd" },
+  TelescopeResultsDiffDelete = { default = true, link = "DiffDelete" },
+  TelescopeResultsDiffUntracked = { default = true, link = "NonText" },
+}
+
+for k, v in pairs(highlights) do
+  vim.api.nvim_set_hl(0, k, v)
+end
+
+-- This is like "<C-R>" in your terminal.
+--     To use it, do `cmap <C-R> <Plug>(TelescopeFuzzyCommandSearch)
+--TODO(conni2461): Make it lua make it awesome
+vim.cmd [==[
```

Comment:
Still todo, Didn't even know that is a thing

Reply:
Yeah, that was (just) for me :)

I've been trying to convert it myself, unsuccessfully...

Probably easiest to use `vim.keymap.set`, but keep the rhs as a vimscript command string.